db/migrate/20191009142101_add_institution_columns_to_institution_programs_archives.rb

```diff
@@ -0,0 +1,40 @@
+class AddInstitutionColumnsToInstitutionProgramsArchives < ActiveRecord::Migration[5.1]
+  disable_ddl_transaction!
+
+  def change
```

Comment:
Same comment as above -- not sure about the safety of disabling the transaction for a schema change (add_column) vs the index.

Reply:
Sounds good.  I moved the index creations to their own PRs